core/enrollments/signals.py

```
@@ -0,0 +1,58 @@
+from django.db.models.signals import post_save, post_delete
+from django.dispatch import receiver
+from django.core.cache import cache
+from .models import Enrollment
+
+
+@receiver(post_save, sender=Enrollment)
+def enrollment_saved(sender, instance, created, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `enrollment_saved`

Add return type annotation: `None`

(ANN201)